drivers/include/ata8520e.h

```diff
@@ -175,24 +176,26 @@ void ata8520e_read_atmel_version(const ata8520e_t *dev, uint8_t *version);
  * @brief Return the Atmel version of the device
  *
  * @param[in]  dev          Pointer to device descriptor
- * @param[out] version      String containing the version.
- *                          The version contains 11 characters.
+ * @param[out] version      String containing the version, must be able to hold
+ *                          11 characters
  */
 void ata8520e_read_sigfox_version(const ata8520e_t *dev, char *version);
 
 /**
  * @brief Return the porting authorization code (PAC) available in the device
  *
  * @param[in]  dev          Pointer to device descriptor
- * @param[out] pac          String containing the pac
+ * @param[out] pac          String containing the pac, must be able to hold 8
```

Comment:
oups, ACK'ed to fast, this is not 8, but 16 here

Reply:
ups, I misread the code, just saw the `SIGFOX_PAC_LENGTH >> 1` but didn't notice that it was the parameter for `fmt_bytes_hex()` and not for `transfer_bytes`... Will fix.

Reply:
but wait, wouldn't it meant that the resulting string size is `17` then? The '_read_pac()` function copies 16 characters into the result string and then puts the `\0` byte in the 17th position (array offset 16) of the result string, right?

Reply:
You are right, I spoke too fast. That's what the driver test application does: see [here](https://github.com/RIOT-OS/RIOT/blob/master/tests/driver_ata8520e/main.c#L61)

Reply:
Perfect, then I will adapt this to the really correct numbers :-)